### ISSUE TITLE
Searching for a job ID that does not exist causes Error 500

### ISSUE THREAD

**Issue by asmaps** [analogbyte/saltobserver#7]
If you enter a job id that does not exist it will cause an internal server error with the following traceback:

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/opt/code/saltobserver/views.py", line 27, in jobs
    function = data.get('fun')
UnboundLocalError: local variable 'data' referenced before assignment
```

**Comment by analogbyte** [comment]
Fixed as of dd85c47 (it's in v0.9.3, too)

**Comment by asmaps** [comment]
Thanks for the quick fix. Works like a charm :-)
